src/utils/response/rawApiResponse.ts

```diff
@@ -22,11 +22,6 @@ export type RawApiResponse = {
 
 export async function createRawApiResponse(promise: Promise<AxiosResponse<any>>): Promise<RawApiResponse> {
   const axiosResponse: AxiosResponse<any> = await promise;
-
-  if (axiosResponse === undefined) {
-    throw new Error('The Api response is unsuccessful');
-  }
-
```

Comment:
@whosshell might have more knowledge if this can be removed or not

Reply:
We had case before when response was undefined, so since this occurred in the past I would keep some check on this.
And whenever it happens we can cover the case.

Reply:
My two cents to this topic, it is not good enough that we only check if the response is undefined and Success is false, we should have a full type guard for the apiResponse to make sure the response looks like what we expect. (https://basarat.gitbook.io/typescript/type-system/typeguard#user-defined-type-guards)

Reply:
rolled back error handling